.github/docs/contributing.md

```diff
@@ -1,7 +1,8 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+We'd love to accept your patches and contributions to existing samples in this project.
+New sample applications are reserved to Google employees only for the time being.
```

Comment:
nit -- alternate suggestion for phrasing here

We welcome **updates and patches** to existing sample applications from all contributors. For **net new sample applications**, however, we're currently only accepting contributions from Google employees to ensure alignment with the project.